source/isaaclab_rl/setup.py

```diff
@@ -34,6 +34,8 @@
     # make sure this is consistent with isaac sim version
     "pillow==11.3.0",
     "packaging<24",
+    # linux aarch 64 requires manual onnxscript installation
```

Comment:
Isn't onnx scripting only needed for RSL-RL? Should we only install it for that case?